_data/shalziekay/shalziekay.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Shalini K
+institution: University of California, Berkeley
+profile_pic: shalini.jpg
+quote: By being brave, not perfect, we get to create and live lives that are authentically, joyfully, messily, and completely ours - Reshma Saujani
```

Comment:
Character count of your quote exceeds the given limit-100. You might want to modify this.